app/controllers/api/cloud_volumes_controller.rb

```diff
@@ -47,5 +47,17 @@ def create_backup_resource(type, id, data)
         {:task_id => cloud_volume.backup_create_queue(User.current_userid, data)}
       end
     end
+
+    def restore_backup_resource(type, id, data)
+      api_resource(type, id, "Restoring backup", :supports => :backup_restore) do |cloud_volume|
```

Comment:
```suggestion
      api_resource(type, id, "Restoring backup for", :supports => :backup_restore) do |cloud_volume|
```

you read it "#{phrase} #{cloud_volume}". So when there are 2 words, we need to add the for... (not perfect, but here we are)

ditto below